modules/feature-list-view/package.json

```diff
@@ -26,11 +26,12 @@
     "@mlvis/mlvis-common": "^1.0.0",
     "d3-scale": "^3.0.0",
     "numeral": "^2.0.6",
+    "prop-types": "^15.7.2",
     "react-d3-axis": "^0.1.1",
-    "prop-types": "^15.7.2"
+    "reselect": "^3.0.1"
   },
   "peerDependencies": {
-    "react": "^16.7.0",
-    "reselect": "^3.0.1"
-  }
+    "react": "^16.7.0"
+  },
+  "gitHead": "0824af3683e1f960379508a01cf599a468dcef88"
```

Comment:
Are these `gitHead` changes intentional? I believe this is not intentional from https://github.com/lerna/lerna/issues/1880#issuecomment-456160357

Reply:
removed gitHead